### PR TITLE
Add method option to sensor.integration

### PR DESCRIPTION
**Pull request by dgomes**
**Description:**

Adds 'method' option to sensor.integration

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#21050

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/sensor.integration.markdown [hunk 1]
```diff
@@ -7,14 +7,14 @@ sidebar: true
 comments: false
 sharing: true
 footer: true
-ha_category: Utility 
+ha_category: Utility, Energy
```
Comment (klaasnicolaas) [inline source/_components/sensor.integration.markdown]:
```suggestion
ha_category: 
  - Utility 
  - Energy 
```

**Review by klaasnicolaas — DISMISSED** [review]

File: source/_components/sensor.integration.markdown (hunk 1 above)
Comment (dgomes) [reply source/_components/sensor.integration.markdown]: Thank you soo much yet again :)

**Comment by tim-devel** [comment]
I have just started using this sensor and was a bit blinded by some of the mathematical terminology. Can you consider adding some more description for the laymen please.  For example what is the difference between trapezoidal and left right and what are the use cases for each

**Comment by dgomes** [comment]
The best explanation is IMHO the wikipedia article which is already linked (explaining Reimann sum)

The use case for each of them depend too much on the user, and it's specific source :(

**Review by balloob — APPROVED** [review]

**Pull request merged**